presentation/voca/src/main/java/com/hilingual/presentation/voca/component/VocaDialog.kt

```diff
@@ -0,0 +1,130 @@
+package com.hilingual.presentation.voca.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Dialog
+import androidx.compose.ui.window.DialogProperties
+import com.hilingual.core.common.extension.noRippleClickable
+import com.hilingual.core.designsystem.component.tag.WordPhraseTypeTag
+import com.hilingual.core.designsystem.theme.HilingualTheme
+import com.hilingual.presentation.voca.R
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+@Composable
+internal fun VocaDialog(
+    onDismiss: () -> Unit,
+    phrase: String,
+    phraseType: List<String>,
+    explanation: String,
+    createdAt: Date,
+    isBookmarked: Boolean,
+    onBookmarkClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    properties: DialogProperties = DialogProperties()
+) {
+    val dateFormat = SimpleDateFormat("yy.MM.dd", Locale.getDefault())
+    val createdAtText = dateFormat.format(createdAt)
+
+    Dialog(
+        onDismissRequest = onDismiss,
+        properties = properties
+    ) {
+        Column(
+            modifier = modifier
+                .fillMaxWidth()
+                .background(
+                    color = HilingualTheme.colors.white,
+                    shape = RoundedCornerShape(12.dp)
+                )
+                .padding(vertical = 28.dp, horizontal = 24.dp)
```

Comment:
```suggestion
                .padding(start = 24.dp, top = 28.dp, end = 24.dp, bottom = 40.dp)
```
p1
피그마 확인해봤을 때 여기 하단 padding값이 40dp인 거 같습니당! 상하좌우로 쪼개서 적용해줘야 할 것 같아용

Reply:
헉 확인 완료 !! 
<img width="196" alt="image" src="https://github.com/user-attachments/assets/3a13dabb-a219-4add-8287-3df9512b6811" />
피그마에 이렇게 나와잇는것만 확인하고 적용해서,.. 실수가 있엇네요 꼼꼼한 지영이 짱 ~~